tutorials/snowplow-cli-mcp/introduction.md

```diff
@@ -16,6 +16,16 @@ The Snowplow CLI MCP (Model Context Protocol) tool integrates Snowplow's data st
 - Creating and validating data structures through conversation
 - AI-powered analysis for strategic tracking plan development
 
+## Demo: Using the Snowplow CLI MCP with Claude Desktop
+
+<iframe
```

Comment:
go ahead and style it to be central, worst case @AH-Avalanche can update it later